@AntoinePrv Can you maybe look at this? Papilo does fail, gcg compiles successfully:
```
FAILED: bin/papilo.exe 
C:\Windows\system32\cmd.exe /C "cd . && %BUILD_PREFIX%\Library\bin\cmake.exe -E vs_link_exe --intdir=papilo\binaries\CMakeFiles\papilo-executable.dir --rc=C:\PROGRA~2\WI3CF2~1\10\bin\100226~1.0\x64\rc.exe --mt=C:\PROGRA~2\WI3CF2~1\10\bin\100226~1.0\x64\mt.exe --manifests  -- C:\PROGRA~1\MICROS~2\2022\ENTERP~1\VC\Tools\MSVC\1429~1.301\bin\HostX64\x64\link.exe /nologo papilo\binaries\CMakeFiles\papilo-executable.dir\__\src\papilo.cpp.obj  /out:bin\papilo.exe /implib:papilo\binaries\papilo.lib /pdb:bin\papilo.pdb /version:0.0 /machine:x64 /INCREMENTAL:NO /subsystem:console  papilo\libpapilo-core.lib  lib\libsoplex.lib  lib\libscip.lib  %PREFIX%\Library\lib\boost_iostreams.lib  %PREFIX%\Library\lib\boost_serialization.lib  %PREFIX%\Library\lib\boost_program_options.lib  %PREFIX%\Library\lib\tbb12.lib  -lpthreads  %PREFIX%\Library\lib\z.lib  %PREFIX%\Library\lib\gmp.lib  %PREFIX%\Library\lib\mpfr.lib  kernel32.lib user32.lib gdi32.lib winspool.lib shell32.lib ole32.lib oleaut32.lib uuid.lib comdlg32.lib advapi32.lib && cd ."
LINK: command "C:\PROGRA~1\MICROS~2\2022\ENTERP~1\VC\Tools\MSVC\1429~1.301\bin\HostX64\x64\link.exe /nologo papilo\binaries\CMakeFiles\papilo-executable.dir\__\src\papilo.cpp.obj /out:bin\papilo.exe /implib:papilo\binaries\papilo.lib /pdb:bin\papilo.pdb /version:0.0 /machine:x64 /INCREMENTAL:NO /subsystem:console papilo\libpapilo-core.lib lib\libsoplex.lib lib\libscip.lib %PREFIX%\Library\lib\boost_iostreams.lib %PREFIX%\Library\lib\boost_serialization.lib %PREFIX%\Library\lib\boost_program_options.lib %PREFIX%\Library\lib\tbb12.lib -lpthreads %PREFIX%\Library\lib\z.lib %PREFIX%\Library\lib\gmp.lib %PREFIX%\Library\lib\mpfr.lib kernel32.lib user32.lib gdi32.lib winspool.lib shell32.lib ole32.lib oleaut32.lib uuid.lib comdlg32.lib advapi32.lib /MANIFEST:EMBED,ID=1" failed (exit code 1104) with the following output:
LINK : warning LNK4044: unrecognized option '/lpthreads'; ignored
   Creating library papilo\binaries\papilo.lib and object papilo\binaries\papilo.exp

LINK : fatal error LNK1104: cannot open file 'papilo\binaries\papilo.exp'

[1346/1346] Linking CXX shared library papilo\binaries\papilo.dll
```

--- a/recipe/meta.yaml
+++ b/recipe/meta.yaml
@@ -194,8 +194,8 @@ outputs:
     test:
       commands:
         - gcg --version
-        - test -d "${PREFIX}/lib/cmake/gcg" # [unix]
-        - test -d "${PREFIX}/include/gcg"   # [unix]
+        - test -d "${PREFIX}/lib/cmake/gcg"  # [unix]
+        - test -d "${PREFIX}/include/gcg"    # [unix]
         - if exist %PREFIX%\\Library\\lib\\cmake\\gcg (exit 0) else (exit 1)  # [win]
         - if exist %PREFIX%\\Library\\include\\gcg (exit 0) else (exit 1)     # [win]
 

--- a/recipe/meta.yaml
+++ b/recipe/meta.yaml
@@ -186,10 +186,11 @@ outputs:
         # run_exports not set for cliquer
         - cliquer   # [unix]
     files:
-      - "lib/libgcg*"
-      - "lib/cmake/gcg"
-      - "bin/gcg"
-      - "include/gcg"
+      - "{{ install_prefix }}/lib/libgcg*"
+      - "{{ install_prefix }}/lib/cmake/gcg"
+      - "{{ install_prefix }}/bin/gcg"
+      - "{{ install_prefix }}/bin/libgcg*"  # [win]
+      - "{{ install_prefix }}/include/gcg"
     test:
       commands:
         - gcg --version

--- a/recipe/meta.yaml
+++ b/recipe/meta.yaml
@@ -194,8 +194,10 @@ outputs:
     test:
       commands:
         - gcg --version
-        - test -d "${PREFIX}/lib/cmake/gcg"
-        - test -d "${PREFIX}/include/gcg"
+        - test -d "${PREFIX}/lib/cmake/gcg" # [unix]
+        - test -d "${PREFIX}/include/gcg"   # [unix]
+        - if exist %PREFIX%\\Library\\lib\\cmake\\gcg (exit 0) else (exit 1)  # [win]
+        - if exist %PREFIX%\\Library\\include\\gcg (exit 0) else (exit 1)     # [win]
 
   - name: zimpl
     version: {{ zimpl_version }}

